fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -3702,6 +3703,20 @@ public void removeFormerIndex(FormerIndex formerIndex) {
         }
     }
 
+    private void clearReadableIndexBuildData(Transaction tr, Index index) {
+        tr.clear(indexRangeSubspace(index).range());
```

Comment:
In theory, it would be good to have a way of testing this. It seems pretty clear to me it's doing the right thing, but it still would be good to have a test that, say, builds an index and makes sure the range space is empty or something, and that the index can still be used.

Reply:
Added `testMarkReadableClearsBuiltRanges`.